src/components/list-item/daikin-list-item.ts

```diff
@@ -0,0 +1,183 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, nothing, unsafeCSS } from "lit";
+import { customElement, property, query } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import tailwindStyles from "../../tailwind.css?inline";
+import "../icon/daikin-icon";
+import type { IconType } from "../icon/daikin-icon";
+
+const cvaListItem = cva(
+  [
+    "flex",
+    "justify-between",
+    "items-center",
+    "gap-2",
+    "w-full",
+    "min-w-max",
+    "min-h-12",
+    "py-3",
+    "pl-4",
+    "pr-3",
+
+    "link-disabled:text-daikinNeutral-200",
+
+    "link-enabled:hover:bg-daikinNeutral-100",
+    "link-enabled:active:bg-daikinNeutral-200",
+
+    "focus-visible:outline",
+    "focus-visible:outline-1",
+    "focus-visible:-outline-offset-1",
+    "focus-visible:outline-daikinBlue-700",
+  ],
+  {
+    variants: {
+      leftIcon: {
+        false: ["pl-4"],
+        true: ["pl-3"],
+      },
+      rightIcon: {
+        false: ["pr-4"],
+        true: ["pr-3"],
+      },
+    },
+  }
+);
+
+/**
+ * The list item component functions as a child element of the list component, and is used to actually list items.
+ *
+ * You can choose between the `button` type, which uses the HTML button tag, and the `link` type, which uses the a tag, and use them according to your needs.
+ *
+ * Hierarchy:
+ * - `daikin-list` > `daikin-list-item`
+ *
+ * @fires click - A retargeted event of a [click event](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event). Suppressed if `disabled` is true.
+ *
+ * @slot - An optional element that is displayed on the right of a list item. e.g. `daikin-checkbox`, `daikin-toggle`
+ *
+ * @example
+ *
+ * ```html
+ * <daikin-list-item label="List item label"></daikin-list-item>
+ * ```
+ */
+@customElement("daikin-list-item")
+export class DaikinListItem extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+  `;
+
+  /**
+   * Label text for the list item.
+   */
+  @property({ type: String, reflect: true })
+  label = "";
+
+  /**
+   * Type of the list item.
+   * If `"link"` is specified, the list item will be rendered as an `<a>` element or `<span>` element (if `disabled` is `true`).
+   */
+  @property({ type: String, reflect: true })
+  type: "button" | "link" = "button";
+
+  /**
+   * Link `href`.
+   * Only used if the `type` is `"link"`.
+   */
+  @property({ type: String, reflect: true })
+  href: string | null = null;
+
+  /**
+   * An icon displayed at the left of the label.
+   */
+  @property({ type: String, reflect: true, attribute: "left-icon" })
+  leftIcon: IconType | null = null;
+
+  /**
+   * Whether the right arrow icon is visible.
+   * If there is content in the slot, it will always be false.
+   */
+  @property({ type: Boolean, reflect: true, attribute: "right-arrow-icon" })
+  rightArrowIcon: boolean = false;
```

Comment:
Please check with the designer whether the icon is fixed or not. If it is variable, it should be something like rightIcon.

otherwise nit. 

```suggestion
  chevron: boolean = false;
```

Reply:
The List is not yet a component that is ready for development, so we will check it as soon as we reach that stage.